docs/README.md

```diff
@@ -15,12 +15,6 @@ Supported authorization providers are:
 
 ## Hiding native login
 
-If you're using a single sign-on provider, you may wish to hide the regular WordPress username and password options from the login screen. This can be configured with the `hide_native` configuration option:
-
-```json
-"sso": {
-	"hide_native": true
-}
-```
+If you're using a single sign-on provider, you may wish to hide the regular WordPress username and password options from the login screen. This can be configured via setting `modules.sso.hide_native` configuration option to `false`
```

Comment:
```suggestion
If you're using a single sign-on provider, you may wish to hide the regular WordPress username and password options from the login screen. This can be configured via setting the `modules.sso.hide_native` configuration option to `false`
```